src/applications/appeals/995/tests/995-supplemental-claim.cypress.spec.js

```diff
@@ -24,7 +24,7 @@ const testConfig = createTestConfig(
     // dataDir: path.join(__dirname, 'data'),
 
     // Rename and modify the test data as needed.
-    dataSets: ['minimal-test'], // , 'maximal-test'],
+    dataSets: ['maximal-test'], // , 'minimal-test'],
```

Comment:
Note to self: Switch this back to `minimal-test`. We can include maximal once the new evidence flow is in place - no need to do it now

Reply:
Noted: Being skipped in CI for now.